integration/airflow/tests/test_macros.py

```diff
@@ -51,6 +51,7 @@ def test_lineage_parent_id():
         task_id="task_id",
         execution_date="execution_date",
         try_number=1,
+        dag_run=mock.MagicMock(run_id="run_id"),
```

Comment:
I think it works without that line, am i wrong?

Reply:
Yes, I think the logic of the macro changed in between, this wasn't fixed.